styles/lazy-styles.css

```diff
@@ -136,4 +136,109 @@ main.error a.button-404  span{
   color: #90f;
   border-bottom: 2px solid #90f;
   line-height: 120%;
-}
\ No newline at end of file
+}
+
+/* Styles for Columns with the "stat" class, and "stat-wrapper" section metadata style */
+div.stat-wrapper {
+  padding: 0;
+}
+
+div.stat-wrapper .columns-wrapper {
+  width: 100%;
+  margin: 0;
+}
+
+div.stat-wrapper .stat {
+  background-color: var(--neutral-carbon);
+  color: var(--neutral-white);
+  padding: var(--spacer-layout-05) var(--spacer-layout-02) var(--spacer-layout-04);
+}
+
+div.stat-wrapper .stat > div {
+  display: flex;
+  align-items: start;
+  column-gap: var(--spacer-element-07);
+}
+
+/* Gradiant line, from purple to red */
+div.stat-wrapper .stat > div > div::before {
+  display: block;
+  content: " ";
+  height: 1px;
+  background-image: linear-gradient(270deg, #fa1e32, #90f);
+}
+
+div.stat-wrapper .stat h1 {
+  font-size: 96px;
+  line-height: 120%;
+  letter-spacing: 0;
+  font-weight: 300;
+  padding: 0;
+  margin: 0;
+}
+
+div.stat-wrapper .stat p {
+  font-size: var(--font-size-16);
+  letter-spacing: .01em;
+  line-height: 160%;
+  font-weight: 400;
+  color: var(--neutral-white);
+  padding: 0;
+  margin: 0 0 var(--spacer-layout-03) 0;
+  overflow-wrap: break-word;
+}
+
+div.stat-wrapper .stat p:last-child {
+  margin-bottom: 0;
+}
+
+/* A single stat element */
+div.stat-wrapper .stat > div > div {
+  margin-bottom: var(--spacer-layout-04);
+  width: 100%;
+}
+
+/* Gray bar which resides below stats */
+div.stat-wrapper .stat::after {
+  display: block;
+  content: " ";
+  height: 1px;
+  border-top: 1px solid #8b8883;
+  padding-top: var(--spacer-element-08);
+}
+
+/* Overriding default column styles b/c the 1st breakpoint for the Stat component is 768px — not 900px */
+@media (min-width: 768px) {
+  .columns.stat > div {
+    display: flex;
+    align-items: center;
+    flex-direction: unset;
+  }
+
+  .columns.stat > div > div {
+    flex: 1;
+  }
+
+  div.stat-wrapper .stat > div > div {
+    margin-left: 0;
+  }
+
+  .columns.stat > div > div:first-of-type {
+    margin-left: unset;
+  }
+
+  div.stat-wrapper .stat {
+    padding: 80px var(--spacer-layout-036) var(--spacer-layout-03);
+  }
+}
+
+/* Desktop styles */
+@media (min-width: 1192px) {
```

Comment:
@amol-anand can confirm but I think `1200px` is the desktop breakpoint being used.

Reply:
`1192` is used in the spec: https://www.figma.com/file/vw24IPXXt4vCh9BNumka9A/Web--Merative-Digital-Design-System-2.0?node-id=258%3A24&t=R9xGLS1DkoVO4xQu-0

<img width="825" alt="image" src="https://user-images.githubusercontent.com/1048207/217294967-bd24dee2-83fd-450a-a053-bd15eb19076d.png">

Reply:
Understood, latest changes are using 1200: https://github.com/hlxsites/merative2/pull/14/files#diff-10e61bb0cdf350cc5fdc389b443350e3952faa739db6564793612e2ea72e391cR83